### ISSUE TITLE
"Used in views" list is broken (/admin/reports/fields/views-fields)

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3318]
## Describe your issue or idea

I discovered this while working on #3316...

### Steps to reproduce (if reporting a bug)

Simply navigate to Reports -> Fields -> Used in views (`/admin/reports/fields/views-fields`)

### Actual behavior (if reporting a bug)

Only the `comment_body` field used in the "Recent comments" view is listed.

### Relevant version/system information (if applicable)

All fields used in all views should be listed.

**Comment by olafgrabienski** [comment]
@klonos I can't reproduce this on a `1.11.0` site. Are you sure on your site are actually any fields (besides the comment body) that are used in (custom) views? On my testing site there are such fields in custom views, and they are listed.

Idea: maybe only the fields of the delivered administrative views don't show up.

**Comment by klonos** [comment]
@olafgrabienski I thought that this page was meant to list fields used in both custom *and* system views (same as the other tab lists fields from content types that come OOTB, as well as custom ones). So I expected to see *all* fields used in *all* views.

Besides, this was in a vanilla installation (you can test in any PR sandbox that does not create any new view). So if only fields from custom views were to be listed, then the `comment_body` field used in the "Recent comments" view should not be there.

**Comment by olafgrabienski** [comment]
@klonos I agree, I'd also expect to see all fields in all views. I was however thinking about a possible reason / motivation why some fields used in Views are not listed. I still think it concerns only the *administrative* views. The `comment_body` field, in contrast, is part of a view for site visitors (a block and a page to list recent comments).

**Comment by klonos** [comment]
Working on this. Here's what I currently have on my local:

![image](https://user-images.githubusercontent.com/2423362/150661194-de80147c-1ddb-4cf3-a9bb-b2a18ebe21ab.png)

(silly test edits to add various fields in a couple of views as fields, filter criteria, sorting criteria, contextual filters - not resulting in any actually useful view)

Once #5090 is in, we could hide the details re specific usage within each view display, and allow people to show/hide them as the like.
